controllers/jenkins/pipelinerun/pipelinerun_controller.go

```diff
@@ -39,12 +49,175 @@ type Reconciler struct {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	_ = context.Background()
-	r.Log.WithValues("pipelinerun", req.NamespacedName).Info("Hello PipelineRun")
+	ctx := context.Background()
+	log := r.Log.WithValues("pipelinerun", req.NamespacedName)
+
+	// get pipeline run
+	var pr devopsv1alpha4.PipelineRun
```

Comment:
It's very easy to make people feel confused about `pr`. Does it mean `Pull Request` or other things?

Reply:
I think the short name is ok, please see [tekton PipelineRun reconciler](https://github.com/tektoncd/pipeline/blob/67b318ba62d9dfee4e9ec639571754f42d9702d7/pkg/reconciler/pipelinerun/pipelinerun.go#L144-L150).